.github/workflows/push_tests.yml

```diff
@@ -16,8 +16,14 @@ jobs:
         with:
           ref: ${{ github.event.pull_request.head.ref }}
       - uses: actions/setup-python@v3
-      - name: check backend
+      - name: Pre-commit checks
         uses: pre-commit/action@v3.0.0
+      - name: Install dependencies
+        run: |
+          pip install -r ./requirements-dev.txt
+      - name: Run mypy
```

Comment:
this can't go in the pre-commit check because for some reason when you invoke `mypy --config-file pyproject.toml` as part of `per-commit run --all-files` it will run mypy across all the files instead of just the package specified in `pyproject.toml`

Reply:
So once we expand the mypy coverage to `cellxgene_schema_cli` directory, do you think we'll still run into this problem? It seems important to resolve, since running `pre-commit` is how we'll all expect to run mypy. If nothing else, move your comment into the yml source. If we can't figure out, make an issue for the future

Reply:
Yes, I believe we'll still run into the problem unless we update this to run across all files in the repo. My guess is that the `--all-files` argument in `pre-commit` overrides the more narrow scanning we specify in `pyproject.toml`, though I can't find any documentation in the pre-commit project actually supporting that. I'll add a comment to both this yaml file and the pre-commit yaml file though, that's a good idea!